entrypoint.sh

```diff
@@ -27,7 +27,9 @@ esac
 JWT=$(cat $JWT_PATH)
 curl -X POST "http://my.dappnode/data-send?key=jwt&data=${JWT}"
 
-exec /nethermind/Nethermind.Runner \
+#Check if the user has added the flag bootnodes
+if [[ "$EXTRA_OPTS" = *"Bootnodes"* ]]; then
```

Comment:
This implementation is oververbose, why not something like

``` 
if [[ $EXTRA_OPTS == *"Bootnodes"* ]]; then
    echo "Bootnodes already set"
else
    echo "Setting bootnodes"
    EXTRA_OPTS="$EXTRA_OPTS --Discovery.Bootnodes=...."
fi
```

Reply:
This way you dont write the entrypoint twice, which may lead to errors

Reply:
Added this way